litgpt/scripts/convert_hf_checkpoint.py

```diff
@@ -491,6 +566,10 @@ def convert_hf_checkpoint(
         # holder to reconstitute the split q, k, v
         qkv_weights = {}
         copy_fn = partial(copy_weights_hf_llama, config, qkv_weights)
+    elif model_name.lower().startswith("qwen2.5"):
```

Comment:
That should be checked before `LLaMAMLP`, because Qwen models have LLaMAMLP class.
The same for `convert_lit_checkpoint.py`.